unzip_content.py

```diff
@@ -287,11 +297,19 @@ def get_icon_path(linkJSON):
                     url = link_data['linkUrl']
                     desktop_objects[id].append_localized_url(lang, url)
 
-    apps_path = os.path.join(UNZIP_DIR, 'apps', 'content.json')
+    apps_path = os.path.join(CONTENT_DIR, 'apps', 'content.json')
```

Comment:
Yes, that was a necessary change, because I now need the categories that have been processed with the extra categories rather than the raw content unzipped from the CMS.  (Previously, it did not matter, and the choice of UNZIP vs. CONTENT was arbitrary.)